client/blocks/simple-site-rename-form/index.jsx

```diff
@@ -0,0 +1,173 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { get, flow, inRange, isEmpty } from 'lodash';
+import Gridicon from 'gridicons';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormButton from 'components/forms/form-button';
+import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+import FormInputValidation from 'components/forms/form-input-validation';
+import ConfirmationDialog from './dialog';
+import FormSectionHeading from 'components/forms/form-section-heading';
+import { requestSiteRename } from 'state/site-rename/actions';
+import { isRequestingSiteRename } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+const SUBDOMAIN_LENGTH_MINIMUM = 4;
+const SUBDOMAIN_LENGTH_MAXIMUM = 50;
+
+export class SimpleSiteRenameForm extends Component {
+	static defaultProps = {
+		currentDomainSuffix: '.wordpress.com',
+		currentDomain: {},
+	};
+
+	state = {
+		showDialog: false,
+		domainFieldValue: '',
+		domainFieldError: '',
+	};
+
+	onConfirm = () => {
+		const { selectedSiteId } = this.props;
+		// @TODO: Give ability to chose whether or not to discard the original domain name.
+		const discard = true;
+
+		this.props.requestSiteRename( selectedSiteId, this.state.domainFieldValue, discard );
+	};
+
+	getDomainValidationMessage( domain ) {
+		const { translate } = this.props;
+
+		if ( isEmpty( domain ) ) {
+			return '';
+		}
+
+		if ( domain.match( /[^a-z0-9]/i ) ) {
+			return translate( 'Domain can only contain letters (a-z) and numbers.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **85** times:
`translate( 'Usernames can only contain lowercase letters (a-z) and numbers.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).